app/views/collections/_document_list.html.erb

```diff
@@ -1,3 +1,5 @@
+<% if documents.empty? %> <h3><%= t('sufia.collection.no_works') %></h3><br>
```

Comment:
Can you put a line break before the `<h3>`?

@mtribone do you think `<h3>` is the right mark-up for that message?